docs/rules/array-bracket-newline.md

```diff
@@ -1,14 +1,11 @@
 ---
-title: array-bracket-newline - Rules
+title: array-bracket-newline
 layout: doc
 edit_link: https://github.com/eslint/eslint/edit/main/docs/src/rules/array-bracket-newline.md
 rule_type: layout
 ---
-<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->
 
-# array-bracket-newline
-
-(fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+<!--FIXABLE-->
```

Comment:
This text is generated by Makefile.js. If we remove it here, it will be lost.

https://deploy-preview-933--eslint.netlify.app/docs/rules/array-bracket-newline

(the same for recommended and suggestions).

Reply:
If the goal is to generate this text during the website build, we should add to this repo some code that generates it.

Reply:
This should be generated by gensite in the ESLint repo. You already identified the bug there that caused this.